### ISSUE TITLE
Bug in padded-blocks rules

### ISSUE THREAD

**Issue by goloroden** [eslint/eslint#1416]
I think that there is a bug in the `padded-blocks` rule. Set it to `never` and try the following code:

```
function () {
  /* foo */
  console.log('bla');
  /* bar */
}
```

Then, `padded-blocks` will complain about two empty lines. Apparently it doesn't work correctly with comments.

**Comment by DenisIzmaylov** [comment]
The same issue for decorators:
```
test/components/connect.spec.js
  50:43  error  Block must not be padded by blank lines  padded-blocks
```
Where lines 50-52 contains:
```javascript
    it('sets displayName correctly', () => {
      @connect(state => state)
      class Container extends Component {
...
```

**Comment by lo1tuma** [comment]
This bug has been fixed. That said, decorators are not supported by the default parser of eslint, so I guess you are using a different parser, please file an issue there.

**Comment by DenisIzmaylov** [comment]
I use eslint 0.23, should I install customer parser for it? Which one?

**Comment by lo1tuma** [comment]
If you use the default parser you should get a syntax error for any decorator. I think the only parser that supports decorators is `babel-eslint`.

**Comment by DenisIzmaylov** [comment]
Oh, sorry. I already use it. But the issue is still actual.

**Comment by lo1tuma** [comment]
Then this is a bug in `babel-eslint`.
